packages/toolpad-core/src/DashboardLayout/DashboardSidebarSubNavigation.tsx

```diff
@@ -287,9 +287,9 @@ function DashboardSidebarSubNavigation({
                       sx={{
                         position: 'absolute',
                         bottom: -18,
-                        left: '50%',
+                        left: '45%',
```

Comment:
I wouldn't change this, this is what is centering the text and it goes together with `transform: 'translateX(-50%)'` below.
And also the font size shouldn't be what is causing the issue and it shouldn't be changed unless necessary.

Can we not just set a minimum width somewhere so that the scrollbar doesn't make things narrower?
The current changes are only trying to disguise the problem but the content is not centered.
I can take a deeper look if it helps.